src/bin/cargo/commands/clean.rs

```diff
@@ -14,6 +14,13 @@ pub fn cli() -> Command {
         .arg_target_triple("Target triple to clean output for")
         .arg_target_dir()
         .arg_manifest_path()
+        .arg(
+            flag(
+                "dry-run",
+                "Display what would be deleted without deleting anything",
+            )
+            .short('n'),
+        )
```

Comment:
And maybe we should bikshed the short alias `-n`?

```suggestion
        .arg_dry_run("Display what would be deleted without deleting anything")
```

Reply:
I'd love to add `-n` as a short alias as there is a strong precedence for it but I agree that should be done separately an we should be using `arg_dry_run`